src/features/website/header/view/HeaderRoot.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Container } from '@mantine/core';
+import { type PropsWithChildren } from 'react';
+
+export function HeaderRoot({ children }: PropsWithChildren) {
+    return (
+        <Box
+            component="header"
+            mx="4vw"
+            mt={16}
+            display="flex"
+            align="center"
+            justify="space-between"
+            bg="var(--whiteblur40, rgba(255, 255, 255, 0.4))"
+            bd="2px solid var(--mantine-color-white)"
+            style={{
+                borderRadius: 24,
+                backdropFilter: 'blur(16px)',
+                position: 'sticky',
+                top: 0,
+                zIndex: 100,
+            }}
+        >
```

Comment:
Всі ті самі зауваження по стилям. 

Але тут є деякі проперті яких немає в mantine - тому можна створити header.module.css і туди перенести те чого немає.

```suggestion
            bg="var(--whiteblur40, rgba(255, 255, 255, 0.4))"
            bd="2px solid var(--mantine-color-white)"
            bdrs={24}
            pos="sticky"
            top={0}
            className={styles.headerRoot}
        >
```